sopel-modules/rat-board.py

```diff
@@ -938,22 +972,33 @@ def _keyfn(rescue):
             continue
         num = len(cases)
         s = 's' if num != 1 else ''
-        t = []
-        t.append("{num} {name} case{s}".format(num=num, name=name, s=s))
+        tmpOutput = []
+        tmpOutput.append("{num} {name} case{s}".format(num=num, name=name, s=s))
         if expand:
             # list all rescues and replace rescues with IGNOREME if only unassigned rescues should be shown and the
             # rescues have more than 0 assigned rats
+            # will also replace every rescue that shouldnt not be shown based on the supplied platform
             # FIXME: should be done easier to read, but it should work. I wanted to stick to the old way it was
             # implemented.
-            templist = (format_rescue(bot, rescue, attr, showassigned, showids, hideboardindexes=False,
-                                      showmarkedfordeletionreason=False) if (
-                (not unassigned) or (len(rescue.rats) == 0 and len(rescue.unidentifiedRats) == 0)) else 'IGNOREME' for
-                        rescue in cases)
+            templist = \
+                (format_rescue(bot, rescue, attr, showassigned, showids, hideboardindexes=False, showmarkedfordeletionreason=False)
+                 if (not unassigned or len(rescue.rats) == 0 and len(rescue.unidentifiedRats) == 0)
+                    and (showAllPlats or rescue.platform in showPlats) else 'IGNOREME'
+                 for rescue in cases)
             formatlist = []
             for formatted in templist:
                 if formatted != 'IGNOREME':
                     formatlist.append(formatted)
-                    t.append(formatted)
+                    tmpOutput.append(formatted)
+            num = len(formatlist) if len(formatlist) != 0 else "No"
+            s = 's' if num != 1 else ''
+            tmpOutput[0] = "{num} {name} case{s}".format(num=num, name=name, s=s)
+        else:
+            tempcount = 0
+            tempcount +=  (1 if (showAllPlats or rescue.platform in showPlats) else 0 for rescue in cases)
+            num = tempcount if tempcount != 0 else "No"
+            s = 's' if num != 1 else ''
```

Comment:
please replace `s` with something more descriptive

Reply:
#WontFix, has been there before + it's descriptive enough